release/downloadIstioCandidate.sh

```diff
@@ -87,6 +92,10 @@ with_arch() {
 
 without_arch() {
   printf "\nDownloading %s from %s ..." "$NAME" "$URL"
+  if ! curl --output /dev/null --silent --head --fail "$ARCH_URL"; then
+    printf "\n%s is not found, please specify a valid ISTIO_VERSION and TARGET_ARCH\n" "$ARCH_URL"
```

Comment:
This should be `$URL` as it is without arch support.
```suggestion
    printf "\n%s is not found, please specify a valid ISTIO_VERSION and TARGET_ARCH\n" "$URL"
```